api/kroxylicious-filter-api/src/main/templates/Kproxy/SpecificFilterArrayInvoker.ftl

```diff
@@ -38,6 +38,7 @@ import org.apache.kafka.common.protocol.ApiMessage;
  * Invoker for KrpcFilters that implement any number of Specific Message interfaces (for
  * example {@link io.kroxylicious.proxy.filter.AlterConfigsResponseFilter}.
  */
+@SuppressWarnings("SwitchStatementWithTooFewBranches")
```

Comment:
I don't understand why this is needed (but I'm only reading the diff of a template used to generate code, so I'm easily confused). Could we make it more localised?

Reply:
same as https://github.com/kroxylicious/kroxylicious/pull/323#discussion_r1213748798